cafebabel/articles/models.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import os
+from hashlib import md5
+from pathlib import Path
+
+from mongoengine import signals
+
+from .. import app, db
+from ..core.helpers import slugify
+from ..users.models import User
+
+
+class Article(db.Document):
+    title = db.StringField(required=True)
+    slug = db.StringField(required=True)
+    language = db.StringField(max_length=2, required=True)
+    category = db.StringField()
+    summary = db.StringField()
+    body = db.StringField(required=True)
+    image = db.StringField()
+    status = db.StringField(default='draft')
```

Comment:
How many statuses do we have? Maybe a `is_draft` boolean field (or even better datetime field) would suffice?